imap_processing/cdf/config/shared/hermes_default_global_cdf_attrs_schema.yaml

```diff
@@ -0,0 +1,278 @@
+CDF_Lib_version:
+  description: > 
+    Version of the CDF Binaries library used to generate the CDF File
+  default: null
+  derived: true
+  required: true # NOT Required in ISTP Guide (Derived)
+  validate: false
+  overwrite: false
```

Comment:
This is unique for HERMES since we're using `spacepy` which relies on the CDF binaries. You should be able to remove this for IMAP since you're using CDFLIB. You might want to re-name or change the description so its more clear this is not the CDF Binaries version.